src/gui/mzroll/controller.cpp

```diff
@@ -19,6 +21,19 @@ Controller::~Controller()
     delete mw;
 }
 
+void Controller::resetMP(QList<QString> keys)
+{
+
+    // we need to change the 'type' from QString to std::string since maven parameters
+    // needs to independent of Qt
+    QList<std::string> _keys;
```

Comment:
@rish9511 Why not create a std list instead of converting a QList to std list before passing it to mavenParameters

Reply:
@shubhra-agrawal  
cant do much about this. resetMp receives the list of keys from peaks/options dialog. and both these dialogs give QList.

Reply:
@rish9511 I meant _keys could have been a std list